README.md

```diff
@@ -6,20 +6,50 @@ Provide common management and troubleshooting tools for the [Rook Ceph](https://
 
 ## Install
 
+Note: This required kubectl [krew](https://krew.sigs.k8s.io/docs/user-guide/setup/install/) to be installed.
+
 To install the plugin, run:
 
   ```kubectl krew install rook-ceph```
 
+  To check plugin version `kubectl krew list` this will list all krew plugin with their current version.
```

Comment:
I think this should be un-indented?

```suggestion
To check plugin version `kubectl krew list` this will list all krew plugin with their current version.
```